### PR TITLE
Update index.html

### PR DESCRIPTION
**Pull request by ghost**
i am a fishy fished fish

### CODE REVIEW

File: thomas/index.html [hunk 1]
```diff
@@ -38,8 +38,12 @@
     <h2>Good game</h2>
     <img onclick="addFish()"  src="http://www.drodd.com/images15/fish8.png" alt="good pic" style=";">
     <p>You have <span id="fishQuantity">0</span> fish</p>
+	<h2> fish.com</h2>
+	<p>onclick="buyapond" pond           15 fish<p>
+	<p> onclick="buyapond"</p>
```
Comment (domdomegg) [inline thomas/index.html]: You don't need this line

File: thomas/index.html [hunk 2]
```diff
@@ -38,8 +38,12 @@
     <h2>Good game</h2>
     <img onclick="addFish()"  src="http://www.drodd.com/images15/fish8.png" alt="good pic" style=";">
     <p>You have <span id="fishQuantity">0</span> fish</p>
+	<h2> fish.com</h2>
+	<p>onclick="buyapond" pond           15 fish<p>
```
Comment (domdomegg) [inline thomas/index.html]: The onclick attribute should be inside the p tag:
`<p onclick="buyapond">Pond: 15 fish<p>`

File: thomas/index.html [hunk 3]
```diff
@@ -51,7 +55,26 @@
         function displayFish() { 
             fishQuantity.innerHTML = fish; 
         }
+	function add fps(){
```
Comment (domdomegg) [inline thomas/index.html]: Can't have spaces in your function name

File: thomas/index.html [hunk 4]
```diff
@@ -51,7 +55,26 @@
...
+	
+	    
+	    function buyapond() {
+	    if(fish>19)
+		    fish = fish - 20
+		    fps = fps + 1
+	    }
+		    
+		    
+
```
Comment (domdomegg) [inline thomas/index.html]: You need a `setInterval` somewhere to call your `addfps` function:
`setInterval(addfps, 1000)`

**Review by domdomegg — CHANGES_REQUESTED** [review]

**Comment by jcxldn** [comment]
:-1: Please do your changes requested by @domdomegg

File: thomas/index.html [hunk 5]
```diff
@@ -51,7 +57,39 @@
         function displayFish() { 
             fishQuantity.innerHTML = fish; 
         }
+	function addfps(){
+	fish = fish + fps
+	}
+	
+	    
+	    function buyapond() {
+	    if(fish>19)
```
Comment (domdomegg) [inline thomas/index.html:66]: Better to use >= 20

**Review by domdomegg — APPROVED** [review]
Still some things to get better but it works now

**Pull request merged**